src/sync.cpp

```diff
@@ -871,12 +891,6 @@ LocalNode* Sync::checkpath(LocalNode* l, string* localpath, string* localname)
     }
     else
     {
-        if (initializing)
-        {
-            delete fa;
-            return NULL;
-        }
-
         LOG_warn << "Error opening file";
```

Comment:
If the "initializing" flag is set, the function will already return NULL due to the new "if" above, so this code won't be reached anymore with that flag set.